### DISCUSSION TITLE
"HEVC decoding is not supported." Message from Windows 11 Edge

### DISCUSSION THREAD

**Discussion by ost91** [w3c/webcodecs#761] [Q&A]
Hi,

I have a question regarding the message below.
"HEVC decoding is not supported."

I'm trying to play video on the web using WebCodecs(use decoding) and WebGPU(use rendering).
Codecs support h264, h265 and work normally in Chrome.
It also works well on MacOS Edge.
However, it does not work properly in Windows 11 Edge.

Windows 11 Edge doesn't support HEVC decoding?

The Edge version I use is 120.0.2210.133.
The decoder config is as follows.

`
const config = {
    codec: 'hvc1.1.6.L120.80',
    codedWidth: 1920,
    codedHeight: 1080,
  };
decoder.configure(config);
`

Please tell me how to decode HEVC codec in Windows Edge.

**Comment by dalecurtis** [comment]
I think this is fixed in newer version of Edge, but you might need to ensure the HEVC codec pack is installed from the Windows store for this to work.

**Reply by ost91** [reply]
It works in Edge canary and Dev, but does not work in Beta and Official released versions.

By the HEVC codec pack you mentioned, are you referring to the HEVC video extension from the Windows Store?
Could you please give me a link?

**Reply by aboba** [reply]
The fix is in Edge 122.  My understanding is that it should "just work" with you having to install the HEVC codec pack yourself.

**Reply by ost91** [reply]
Um...
I checked with Edge122 version and also installed HEVC Video extensions from the Windows Stroe.
But it's still not working on your example page.(https://w3c.github.io/webcodecs/samples/video-decode-display/)
I selected WebGPU and H.265 and clicked the start button, but the message "NotSupportedError: HEVC decoding is not supported." still appears.
I leave my Edge version and edge://gpu information.
If you think you know how to solve it, please let me know. thank you.

![image](https://github.com/w3c/webcodecs/assets/26373601/91550502-ba43-4909-90b6-7bb77fa3859b)

![image](https://github.com/w3c/webcodecs/assets/26373601/6d7f1673-bdd2-402d-92dc-42947c418af4)
![image](https://github.com/w3c/webcodecs/assets/26373601/d6384ff2-0487-4f20-a51a-dfc5c1f4157f)
![image](https://github.com/w3c/webcodecs/assets/26373601/aef740c3-fcf3-45db-bda4-f825e9920ea7)

**Comment by StaZhu** [comment]
I think for edge, the browser still have to wait for the experiment result to re-enable `VDAVideoDecoder` which fixes your issue.

```
msedge.exe --disable-features=msCodecFallbackUsingMFClear --enable-features="msD3D11VdaHybridVideoDecoding,msDXVAVideoDecoding,msHEVCWebCodecsEnabled"
```
maybe this works for you. 

More info: https://bugs.chromium.org/p/chromium/issues/detail?id=1508276#c9
